src/main/kotlin/org/jetbrains/research/ml/ast/cli/Configuration.kt

```diff
@@ -30,7 +30,7 @@ data class CliTransformation(
 
 @Serializable
 data class Configuration(
-    @SerialName("n transformations")
+    @SerialName("n apply")
     val numTransformations: Int,
```

Comment:
let's also rename numTransformation, not only its serial name. The same about num